client/common/sass/components/_blog-card.sass

```diff
@@ -1,12 +1,23 @@
 .blog-card
 	border-top: 1px solid
 	border-color: $black
+	padding-top: 2rem
 
 	&--info
 		margin-top: 12px
 		margin-bottom: 0
 		transition: margin-bottom 0.09s ease-in
 
+	&--inner
+		transition: transform 0.15s ease-in-out
+		&:hover
+			transform: translateY(-0.5rem)
+			cursor: pointer
```

Comment:
I did notice that when Javascript is disabled, there are parts of the card where the cursor is a pointer but it is not clickable. I'm not sure how to fix this, or if it's even worth doing, but it was something I noticed.

Reply:
Yes. I did mention this. I don't know any easy way of not doing it either.